src/olympia/amo/mail.py

```diff
@@ -17,30 +17,39 @@ class DevEmailBackend(BaseEmailBackend):
     `settings.SEND_REAL_EMAIL` is disabled.
 
     BUT even if `settings.SEND_REAL_EMAIL` is disabled, if the targeted
-    email address is in the `settings.EMAIL_QA_ALLOW_LIST` list,
-    the email will be sent.
+    email address is in `dev_email` group users, the email will be sent.
     """
 
+    @staticmethod
+    def force_send_mail_group():
+        return 'Force Send Mail'
```

Comment:
Nit: I don't think we need an email, you can make it a simple property instead
```suggestion
    force_send_mail_group = 'Force Send Mail'
```

Reply:
Done.